src/lib/validation/recipient.ts

```
@@ -39,11 +41,12 @@ export async function validateAndResolveRecipient(
             if (!isValidPeanutUsername) {
                 throw new RecipientValidationError('Invalid Peanut username')
             }
-            const address = await resolveFromEnsName(`${recipient}.${JUSTANAME_ENS}`)
+            const user = await usersApi.getByUsername(recipient)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)